bed_maker.py

```diff
@@ -113,22 +103,19 @@ def main():
         print("Determined path to chrom.sizes asset: {}".format(chrom_sizes))
         # define a target for temporary bw files
         temp_target = os.path.join(sample_folder, file_id + ".bw")
+        pm.clean_add(temp_target)
```

Comment:
@michalstolarczyk shouldn't we clean the intermediate files directory after producing the bed files (running `cmd`)?

Reply:
the intermediate files can be added to the cleanup list anytime and will be cleaned upon successful pipeline run. This guide explains this concept well: https://pypiper.readthedocs.io/en/latest/clean/#cleaning-up-intermediate-files

Reply:
Right, I forgot that the intermediate files are cleaned upon pipeline completion.